mozilla/bedrock
# [fix bug 1422094] Updates to State of Mozilla 2016

## Description
- Converts static financial chart to a `<table>`.
- Tweaks responsive section sizes to suit tabular data a bit better. The table is never going to make great reading on mobile, but I made it as accessible as I could using `overflow: scroll;` to preserve it's minimum size.
- Removes `som-2016` switch.
- Other minor tweaks / amends.

## Issue / Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1422094

## Testing
- Check responsiveness?
- Check for typos?
